packages/terra-site/src/examples/dynamic-grid/Dashboard.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import DynamicGrid from 'terra-dynamic-grid';
+
+import Card from './Card';
+
+const layouts = [{
+  'grid-template-columns': '300px 1fr 300px',
```

Comment:
nit: Switching to use percentages here would make these fluid and avoid a horizontal scrollbar on smaller widths on the doc site. 
`300px 1fr 300px` => `30% 1fr 30%`